motor_encoder_node:
  ros__parameters:
    speed: 1600 #int
    pos_init: 1100 
    direction_flag: -1
